apps/settings/build/settings.build.jslike

```diff
@@ -15,6 +15,7 @@
     'shared/apn_helper': 'empty:',
     'shared/async_storage': 'empty:',
     'shared/bluetooth_helper': 'empty:',
+    'modules/bluetooth/bluetooth_context': 'empty:',
```

Comment:
This is not required because currently `modules/bluetooth/bluetooth_context` is not listed as a dependent module in other modules.